core-build/gulp-core-build/src/tasks/JestReporter.ts

```diff
@@ -6,15 +11,50 @@ const DEFAULT_REPORTER: any = require('jest-cli/build/reporters/default_reporter
  * rush does not get confused.
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: any): any }) {
+class JestReporter extends (DEFAULT_REPORTER as { new (globalConfig: Jest.GlobalConfig): any }) {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public constructor(globalConfig: any) {
+  private _globalConfig: Jest.GlobalConfig;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public constructor(globalConfig: Jest.GlobalConfig) {
     super(globalConfig);
+    this._globalConfig = globalConfig;
+
+    this._err = this._out;
   }
 
   public log(message: string): void {
     process.stdout.write(message + '\n');
   }
+
+  public onRunComplete(contexts: Set<Jest.Context>, results: Jest.AggregatedResult): void {
+    super.onRunComplete(contexts, results);
+
+    let outputPath: string | undefined = undefined;
+    let filename: string | undefined = undefined;
+
+    for (const reporter of (this._globalConfig.reporters as (string | ReporterConfig)[])) {
+      const reporterConfig: ReporterConfig | undefined = reporter as ReporterConfig;
```

Comment:
Shouldn't the type on line 36 be `(ReporterConfig | undefined)[]` if this is the real type?

Reply:
ReporterConfig is the tuple type. String is also valid. You could hypothetically see a config along the lines of:
```
    reporters: [ 'reporter1', 'reporter2', [ 'reporter3', { reporterArg1: 'arg1Value' } ] ]
```